serverless/src/getConcepts/handler.js

```diff
@@ -349,18 +352,54 @@ export const getConcepts = async (event, context) => {
     performanceMetrics.totalTime = (endTime - startTime).toFixed(2)
     console.log('get concepts performance=', JSON.stringify(performanceMetrics))
 
-    return {
-      statusCode: 200,
-      body: responseBody,
-      headers: {
-        ...defaultResponseHeaders,
-        'Content-Type': `${contentType}; charset=utf-8`,
-        'X-Total-Count': totalConcepts.toString(),
-        'X-Page-Number': pageNum.toString(),
-        'X-Page-Size': pageSize.toString(),
-        'X-Total-Pages': totalPages.toString()
+    const SIZE_THRESHOLD = 5 * 1024 * 1024 // 5MB in bytes
```

Comment:
add a comment that API Gateway has a hard limit of responses at 6MB

Reply:
Done